draft-ietf-kitten-krb-spake-preauth-04.xml

```diff
@@ -503,7 +492,7 @@ KEY_USAGE_SPAKE_FACTOR                  65
         the IANA "Kerberos SPAKE Groups" registry created by this
         document.</t>
         <t>Compose a pepper string by concatenating the string "SPAKEsecret"
-        and the group number as a big-endian four-byte unsigned binary
+        and the group number as a big-endian four-byte two's complement binary
```

Comment:
We have negative group values now?

Reply:
We have from the start; see the IANA information.  Negative numbers are private/experimental use.  Same situation for enctype numbers (see RFC 3961 section 11).